### PR TITLE
TBS OKTA-456193 Updates redirect auth use cases

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Adds EML to redirect auth for sample apps. also, updates multifactor auth section.
- **Is this PR related to a Monolith release?** late merge

### Netlify link
* https://6256dc844ad85a2cfdecc9c2--reverent-murdock-829d24.netlify.app/docs/guides/sampleapp-oie-redirectauth/nodeexpress/main/

### Resolves:
* [OKTA-456193](https://oktainc.atlassian.net/browse/OKTA-456193)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md [hunk 1]
```diff
@@ -128,37 +128,53 @@ You can modify the Application's Sign-On Policy to require the user to have a se
...
+5. Fill out the requested phone authentication information and verify your phone with a code.
+6. Under **Set up optional**, click **Set up later**. You are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
+7. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
+
+## Self-service Password Recovery
+
+### Email magic link password recovery flow
 
-## Authenticator recovery
+> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:148]: I love this note for making note of a default setting. I'm going to steal it. :)
```suggestion
> **Note:** In your org, password reset is configured by default to be initiated with an email. The steps in this section assume that you haven't changed that default configuration.
```

**Review by susanharper-okta — APPROVED** [review]
LGTM. Not even one nit. 🚀

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:148]: :)

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md [hunk 2]
```diff
@@ -101,11 +101,13 @@ This section walks you through enabling self-service enrollment for the Sign-In
 
 This section walks you through the self-service enrollment steps for a new user.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:102]: Generally speaking, these changes detail using Okta email with OTP. The description of the related PR (EML: Update Redirect auth use cases article to add magic link coverage) discusses the magic link flow where the user clicks on a link in the email to complete the verification. Am I reading this right?

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md [hunk 3]
```diff
@@ -128,37 +130,55 @@ You can modify the Application's Sign-On Policy to require the user to have a se
...
-6. Select **Password + Another factor** and click **Save**.
+4. Scroll down to the **User authentication** section and click **Edit**.
+5. From the **Authentication policy** dropdown menu, select **Any two factors** and click **Save**.
 
 ### Try multifactor authentication
 
 <StackSnippet snippet="tryenrollin" />
 
-3. Enter the credentials of the user that you enrolled earlier. The Set up authentications page appears, which prompts you to set up either the Okta Verify or the Phone authenticator.
+3. Enter the credentials of the user that you enrolled earlier. The Set up security methods page appears, which prompts you to set up either the Okta Verify app or the Phone authenticator.
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:140]: Consider rewording to: "Enter the credentials of the user that you enrolled with earlier. The Set up security methods page appears, prompting you to set up either the Okta Verify app or the Phone authenticator."

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md [hunk 4]
```diff
@@ -128,37 +130,55 @@ You can modify the Application's Sign-On Policy to require the user to have a se
...
+1. Enter a new password.
+1. After you enter the new password successfully, you are prompted for the additional phone authentication that you set up in [Enable multifactor authentication](#enable-multifactor-authentication). Then, you are redirected to the <StackSnippet snippet="applang" inline /> welcome page.
 1. Click <StackSnippet snippet="tryenrollout" inline /> to sign out of the <StackSnippet snippet="applang" inline /> app.
 
-### Recovery with Okta Verify
+### Okta Verify recovery flow
 
 In addition to recovering your password with an email, you can add Okta Verify as a recovery option.
 
+<!-- Needs a link to verify setup as recovery option -->
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:180]: Does this comment refer to a change that needs to happen to the content post publish?

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md [hunk 5]
```diff
@@ -101,11 +101,13 @@ This section walks you through enabling self-service enrollment for the Sign-In
 
 This section walks you through the self-service enrollment steps for a new user.
 
+> **Note:** Enrollment uses the **Registration - Email Verification** email template. You can edit this template. See [Edit a default email template](/docs/guides/custom-email/main/#edit-a-default-email-template).
+
 <StackSnippet snippet="tryenrollin" />
 
 3. In the Okta Sign-In Widget, click **Sign up** just below the **Forgot password?** link.
 4. Enter the requested information, and then click **Sign Up**.
-5. Set up the Email, Password, and Security Question factors. Don't set up any other factors.
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:108]: @robertnicolo-okta Good question. It's "factors" in the UI.

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md (hunk 4 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md]: Yes.

File: packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md:102]: You're absolutely right. Fixing.

**Pull request merged**